test/rest-adapter.test.js

```diff
@@ -200,6 +200,32 @@ describe('RestAdapter', function() {
       });
     });
 
+    describe('getArgByName()', function() {
+      var anArg = [{
+        arg: 'argName1',
+        type: String,
+      }, {
+        arg: 'argName2',
+        type: String,
+      }];
+
+      it('should find the first arg', function() {
+        var method = givenRestStaticMethod({accepts: anArg});
+        expect(method.getArgByName('argName1',
+          ['firstArg', 'secondArg'])).to.equal('firstArg');
```

Comment:
Breaking the line in the middle of argument list make the code difficult to follow. Here is a better solution that we are already using in other parts of our codebase:

```js
expect(method.getArgByName('argName1', ['firstArg', 'secondArg']))
  .to.equal('firstArg');
```